config/plugins/interactive_environments/jupyter/config/allowed_images.yml.sample

```diff
@@ -0,0 +1,14 @@
+# This file lists acceptable images to allow runing.
+#
+# This allows you, the admin, to create multiple flavours
+# for your users to run. E.g. maybe you need a geosciences flavour,
+# you can create the image based on our default image and add the
+# appropriate `apt-get/pip install` statements.
+---
+-
+    image: bgruening/docker-jupyter-notebook:16.01
```

Comment:
Any idea how we can create backup images. Dockerhub is since month very unreliable and slow so I switched to https://quay.io for many images and try to offer both services. Does a comma separated list makes sense or is this just to confusing and we should offer a alternative image as comment?